plugins/woocommerce/src/Internal/ReceiptRendering/Templates/order-receipt-css.php

```diff
@@ -30,3 +30,6 @@
 	background-image: url("data:image/svg+xml;base64,<?php echo $data['payment_info']['card_icon']; ?>");
 }
 <?php } ?>
+.order-status-failed {
+	color: #FF0000;
+}
```

Comment:
Would it be worth wrapping this is a conditional check of `$data['is_order_failed']`, similar to the card icon conditional above?

Reply:
Good suggestion, I think I'll remove this new CSS code.